libevmasm/Assembly.cpp

```diff
@@ -506,83 +506,81 @@ Json::Value Assembly::assemblyJSON(std::map<std::string, unsigned> const& _sourc
-	if (_json.isMember("sourceList"))
+	if (_json.contains("sourceList"))
 	{
 		solAssert(_level == 0);
 		solAssert(_sourceList.empty());
-		for (Json::Value const& sourceName: _json["sourceList"])
+		for (Json const& sourceName: _json["sourceList"])
 		{
 			solRequire(
-				std::find(parsedSourceList.begin(), parsedSourceList.end(), sourceName.asString()) == parsedSourceList.end(),
+				std::find(parsedSourceList.begin(), parsedSourceList.end(), sourceName.get<std::string>()) == parsedSourceList.end(),
 				AssemblyImportException,
 				"Items in 'sourceList' array are not unique."
 			);
-			parsedSourceList.emplace_back(sourceName.asString());
+			parsedSourceList.emplace_back(sourceName.get<std::string>());
 		}
 	}
 
-	solRequire(_json.isMember(".code"), AssemblyImportException, "Member '.code' is missing.");
-	solRequire(_json[".code"].isArray(), AssemblyImportException, "Member '.code' is not an array.");
-	for (Json::Value const& codeItem: _json[".code"])
-		solRequire(codeItem.isObject(), AssemblyImportException, "The '.code' array contains an item that is not an object.");
+	solRequire(_json.contains(".code"), AssemblyImportException, "Member '.code' is missing.");
+	solRequire(_json[".code"].is_array(), AssemblyImportException, "Member '.code' is not an array.");
+	for (Json const& codeItem: _json[".code"])
+		solRequire(codeItem.is_object(), AssemblyImportException, "The '.code' array contains an item that is not an object.");
 
 	result->importAssemblyItemsFromJSON(_json[".code"], _level == 0 ? parsedSourceList : _sourceList);
 
-	if (_json.isMember(".auxdata"))
+	if (_json.contains(".auxdata"))
 	{
-		solRequire(_json[".auxdata"].isString(), AssemblyImportException, "Optional member '.auxdata' is not a string.");
-		result->m_auxiliaryData = fromHex(_json[".auxdata"].asString());
+		solRequire(_json[".auxdata"].is_string(), AssemblyImportException, "Optional member '.auxdata' is not a string.");
+		result->m_auxiliaryData = fromHex(_json[".auxdata"].get<std::string>());
 		solRequire(!result->m_auxiliaryData.empty(), AssemblyImportException, "Optional member '.auxdata' is not a valid hexadecimal string.");
 	}
 
-	if (_json.isMember(".data"))
+	if (_json.contains(".data"))
 	{
-		solRequire(_json[".data"].isObject(), AssemblyImportException, "Optional member '.data' is not an object.");
-		Json::Value const& data = _json[".data"];
+		solRequire(_json[".data"].is_object(), AssemblyImportException, "Optional member '.data' is not an object.");
+		Json const& data = _json[".data"];
 		std::map<size_t, std::shared_ptr<Assembly>> subAssemblies;
-		for (Json::ValueConstIterator dataIter = data.begin(); dataIter != data.end(); dataIter++)
+		for (auto const& [dataItemID, value] : data.items())
 		{
-			solAssert(dataIter.key().isString());
-			std::string dataItemID = dataIter.key().asString();
-			Json::Value const& dataItem = data[dataItemID];
-			if (dataItem.isString())
+			Json const& dataItem = data[dataItemID];
```

Comment:
Isn't this just ``value``?

Reply:
Yes! I was sure that I changed already exactly this :D